pyramid_sna/oauth2.py

```
@@ -78,6 +78,6 @@ def get_user_info(info_uri, access_token):
     response = requests.get(info_uri, headers=headers)
 
     if response.status_code != 200:
-        return HTTPUnauthorized(response.text)
+        raise HTTPUnauthorized(response.text)
```

Can you explain why is it better to raise the HTTPUnauthorized response instead of returning it?